book-service-rest/src/main/java/com/example/Library/services/impl/LoginAttemptService.java

```diff
@@ -0,0 +1,62 @@
+package com.example.Library.services.impl;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+
+import org.springframework.stereotype.Service;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+
+@Service
+public class LoginAttemptService {
+
+    private final int MAX_ATTEMPT = 3;
+    private LoadingCache<String, Integer> attemptsCache;
+
+    public LoginAttemptService() {
+        super();
+        attemptsCache = CacheBuilder.newBuilder().
+                expireAfterWrite(5, TimeUnit.MINUTES).build(new CacheLoader<String, Integer>() {
+                    public Integer load(String key) {
+                        return 0;
+                    }
+                });
+    }
+
+    public void loginSucceeded(String key) {
+        attemptsCache.invalidate(key);
+    }
+    public ArrayList<String> getAllBlockedUsers(){
```

Comment:
please fix the spacing between the methods, it should be 1 new empty line between declared methods